docs/application/dotnet/guides/nui/quickpanelclient.md

```diff
@@ -0,0 +1,105 @@
+# Quickpanel client
+
+Quickpanel is a library to control the Quickpanel service window that shows notifications and system setup widgets. You can use Quickpanel to get various information from the Quickpanel service window. For example, media player application needs to close the Quickpanel service window during playback of video. In this case, you can use the Quickpanel client.
+
+## Prerequisites
+To use the methods of the Quickpanel API, use `Tizen.NUI.WindowSystem` in your application:
+```
+using Tizen.NUI.WindowSystem;
+```
+
+## Create QuickPanel handle
+After you have created the main window of your application, call `QuickPanelClient()` with a `TizenShell` and window to create the handler:
+```
+private void onInitialize()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    // Do something with qpClient
+}
+```
+
+## Show or hide Quickpanel window
+To show or hide the Quickpanel service window when your application’s window is activated, use the following code:
+```
+public void qpShowHide()
+{
+    Window window = NUIApplication.GetDefaultWindow();
+    Shell.TizenShell tzShell = new Shell.TizenShell();
+    Shell.QuickPanelClient qpClient = new Shell.QuickPanelClient(tzShell, window, Shell.QuickPanelClient.Types.SystemDefault);
+
+    qpClient.Hide();
+
+    // Do something without Quickpanel
+
+    qpClient.Show();
+}
+```
+
+## Get state of Quickpanel window
+The Quickpanel window supports three states: `VisibleState`, `Orientation`, and `ScrollableState`. To get the current status for each of the Quickpanel service window state, use the following codes:
+```
+// After qpClient is created
+
+VisibleState visible = qpClient.Visible;
+Window.WindowOrientation orientation = qpClient.Orientation;
+ScrollableState scrollable = qpClient.Scrollable;
+```
+The following are the description of states:
+- **VisibleState**: The visible state of Quickpanel service window. There are three possible values for VisibleState: `Unknown`, `Shown`, or `Hidden`.
+- **Orientation**: The orientation of Quickpanel service window. There are four possible orientation angles: `Angle_0`, `Angle_90`, `Angle_180`, or `Angle_270`.
```

Comment:
- **VisibleState**: The visible state of Quickpanel service window. The three possible values for VisibleState are `Unknown`, `Shown`, or `Hidden`.
- **Orientation**: The orientation of Quickpanel service window. The four possible orientation angles are `Angle_0`, `Angle_90`, `Angle_180`, or `Angle_270`.